opts.go

```diff
@@ -66,6 +66,7 @@ type Options struct {
 	IncludeEvents       bool     `json:"includeEvents"`
 	Prompt              bool     `json:"prompt"`
 	CredentialOverrides []string `json:"credentialOverrides"`
+	CredentialContext   []string `json:"credentialContext"`
```

Comment:
Let's make the field plural, but keep the `json` tag the same so that it is compatible with the SDK server. Maybe add a comment that this difference is intentional.

Also, it is possible to add a test for this?

Reply:
:+1:

Reply:
Done